server/@types/activitiesAPI/index.d.ts

```diff
@@ -312,10 +312,35 @@ export interface components {
        */
       id: number
       /**
-       * @description The reason codes- ABS, ACCAB, ATT, CANC, NREQ, SUS, UNACAB, REST
+       * @description The reason codes- SICK, REFUSED, NREQ, REST, CLASH, OTHER, SUSP, CANC, ATT
        * @example ATT
```

Comment:
The attendance outcome codes don't seem to match the list here?  https://dsdmoj.atlassian.net/wiki/spaces/SAA/pages/4337861296/Attendance+Reasons+-+Mapping+to+NOMIS - might also be worth a talk through tomorrow?

Reply:
Corrected